### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by maryammdot**

### CODE REVIEW

File: staff/maryam-malek/mailchimp/index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                <a class="nav__element" href="support">Support</a>
+                <a class="nav__element" href="blog">Blog</a>
+                <div class="nav__element--input">
+                    <input class="search__input" type="text" name="search" placeholder="Search...">
+                </div>
+            </nav> 
+        </div>
+        <img class="header__image header__image--monkey" src="./images/easy-email-newsletters-banner-png8.png" alt="Monkey banner">
+        <div class="header__info">
+            <img class="header__image header__info__letters" src="./images/easy-email-newsletters-text.png" alt="Easy email Newsletters">
```
Comment (anasanjuan) [inline staff/maryam-malek/mailchimp/index.html]: is it correct to concatenate three words like this? maybe info__letters ?

File: staff/maryam-malek/mailchimp/index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            <section class="main__section main__section--denim">
+                <img class = "main__image" src="./images/home/denim2.jpg" alt="Raleigh Denim image">
+                <h5>Customer Story</h5>
+                <h2>Raleigh Denim</h2>
+                <p class="main__text">What's so special about handcrafted blue jeans? We visited the Raleigh Denim workshop to find out.</p>
+                <br>
+                <button class="button button__main">
+                    Learn more 
+                    <div class="div__img">
+                        <img class="image__arrow" src="./images/arrows.png" alt="Aroow">
```
Comment (anasanjuan) [inline staff/maryam-malek/mailchimp/index.html]: I think there is a typo in alt, it  should be Arrow

File: staff/maryam-malek/mailchimp/index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            <p class="header__text">MailChimp helps you design email newsletters, share them on social networks, integrate with services you already use, and track your results. It's like your own personal publishing platform</p>
+            <button class="button button__big header__info__button grad">Sign Up Free</button>
+        </div>
+    </header>
+    <div class="div__main">
+        <div class=""></div>
+        <!-- <img class="div__main__image div__main__image--left" src="./images/noisepattern.png" alt="noise pattern"> -->
+        <main class="main">
+            <h1 class="main__title">What's New</h1>
+            <section class="main__section main__section--templates">
```
Comment (anasanjuan) [inline staff/maryam-malek/mailchimp/index.html:43]: justify content to have space between and around the main__sections

File: staff/maryam-malek/mailchimp/index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                <!-- <button class="button button__main">Learn more </button> -->
+            </section>
+            <section class="main__section main__section--unfurlr">
+                <img class = "main__image" src="./images/home/unfurlr-thumb.jpg" alt="unfurlr-thumb image">
+                <h5>App Spotlight:</h5>
+                <h2>Unfurlr</h2>
+                <p class="main__text">Unfurlr shows you the destination of shortened URLs before you click, so you can avoid phishing attacks and other nefarious links.</p>
+                <br>
+                <button class="button button__main">
+                    Learn more
```
Comment (anasanjuan) [inline staff/maryam-malek/mailchimp/index.html:78]: make the height of the button bigger and padding to the right

File: staff/maryam-malek/mailchimp/style.sass [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+        grid-area: 1 / 1 / 2 / 2
+        padding-left: 15%
+        max-width: 100%
+        max-height: 100%
+    &__section
+        display: flex
+        align-items: stretch
+        flex-direction: column
+        justify-content: space-between
+        &--templates
```
Comment (anasanjuan) [inline staff/maryam-malek/mailchimp/style.sass:102]: maybe here you can use the extend method so all the main-templates have the same padding, max-width and max-height and you can have a more DRY sass

**Comment by anasanjuan** [comment]
Nice that you apply Sass and BEM in your mailchimp!  :)

File: staff/maryam-malek/mailchimp/index.html (hunk 1 above)
Comment (maryammdot) [reply staff/maryam-malek/mailchimp/index.html]: You are right, I would have to create new block.

File: staff/maryam-malek/mailchimp/index.html (hunk 3 above)
Comment (maryammdot) [reply staff/maryam-malek/mailchimp/index.html:43]: I've organized main with grid, so I think in this case I can not do justify content with space between or arround. But I've given some space with padding.

**Pull request merged**